### DISCUSSION TITLE
Needs documentation of hosting without docker

### DISCUSSION THREAD

**Discussion by Emojigit** [plausible/analytics#1259] [Self-Hosted Support]
I have only one server which disallows me to host a docker site (since my webserver uses 80 and 443). also, if I host it on docker, I cannot use my SSL cert easily. So I think documentation of hosting without docker is required.

BTW please ~~hell~~ tell me where is the right place to say this since I think this is not the best place.

**Comment by natrius** [comment]
I would love more information on that matter because i simply don't want to use Docker at all. One advantage of Matomo (even if bloated) for a small site is, i can simply FTP the stuff on my webserver and everything is working. Thinking about https://matomo.org/faq/on-premise/installing-matomo/ this (had to look it up, just to be sure). 

I'm currently aware of https://plausible.io, https://github.com/milesmcc/shynet, https://66analytics.com/ and finally https://matomo.org/ and only 66analytics and matomo offers this kind of 'easy' installation. (and without docker).  (Sorry if i mixed up 2 different things here)
